### DISCUSSION TITLE
Replace "Prev" and "Next"

### DISCUSSION THREAD

**Discussion by KillWolf** [ddnexus/pagy#441] [Q&A] [answered]
Hey, 

I am trying to edit the "Prev" and "Next" button to be just arrows. I have looked through the Documentation, but I haven't been able to find anything that support this.

Can you point me in the right direction?

I should mention I am using the bootstrap nav. 

Thanks! :D

**Comment by ddnexus** [comment, marked as answer]
"prev" and "next" are 2 entries in the [i18n dictionary](https://github.com/ddnexus/pagy/blob/9b28f5814a9358b1dd7de177aa1a9f098b9d7562/lib/locales/en.yml#L11-L12). Just override them.

**Reply by ddnexus** [reply]
@KillWolf Was the question answered?

**Reply by ddnexus** [reply]
Assuming it's answered

**Reply by ddnexus** [reply]
See [Customize the dictionary](https://ddnexus.github.io/pagy/docs/how-to/#customize-the-dictionary)
